### PR TITLE
Fix errors in tests

### PR DESCRIPTION
**Pull request by commander-bryan**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -105,6 +104,7 @@
     "react-apollo": "2.3.3",
     "react-aria-offcanvas": "^1.3.6",
     "react-dom": "16.8.6",
+    "react-ga": "^2.4.1",
```
Comment (adam-wheatley) [inline package.json]: why do we need this?

File: src/js/PrintView/index.jsx [hunk 2]
```diff
@@ -1,4 +1,132 @@
-import { enhance } from '..';
+import { graphql } from 'react-apollo';
```
Comment (adam-wheatley) [inline src/js/PrintView/index.jsx]: The cycle lint error, is due to the print route generator function being used i think, we will be extracting that anyway, which would save us duplicating all of this

File: package.json (hunk 1 above)
Comment (commander-bryan) [reply package.json]: Its a peer dependency of react-graphql-errors

File: src/js/PrintView/index.jsx (hunk 2 above)
Comment (commander-bryan) [reply src/js/PrintView/index.jsx]: Yeah, I've got a separate ticket for fixing all the linting errors that have been ignored. Quite a lot of them

File: package.json (hunk 1 above)
Comment (adam-wheatley) [reply package.json]: i see

File: src/js/PrintView/index.jsx (hunk 2 above)
Comment (adam-wheatley) [reply src/js/PrintView/index.jsx]: so..whats the purpose of this change if its not  to fix a linting error

Comment (commander-bryan) [reply src/js/PrintView/index.jsx]: The import wasnt working during tests and I couldn't figure out why. Defining everything required by the test in the test solved the problem

Comment (adam-wheatley) [reply src/js/PrintView/index.jsx]: I refer you to my initial comment to fix your issue rather than duplicating all this code.

**Pull request merged**